### PR TITLE
Update iota-token article

### PR DESCRIPTION
**Pull request by vivekjain23**
Updating exchanges listing IOTA in the wiki and fixing the [issue](https://github.com/iotaledger/iota-wiki/issues/1582)

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [X] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [X] I have performed a self-review of my own changes
- [X] I have made sure that added/changed links still work
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docs/get-started/introduction/iota/iota-token.md [hunk 1]
```diff
@@ -72,8 +72,11 @@ using so you don't get tricked into selling/buying under/over the current market
...
 
-- [Binance.US](https://www.binance.us/en/home)
 - [Binance](https://www.binance.com)
+- [Binance.US](https://www.binance.us/en/home)
+- [OKX](https://www.okx.com/)
+- [Gate.io](https://www.gate.io/)
+- [HTX](https://www.htx.com/)
 - [Bitfinex](https://www.bitfinex.com/)
 - [Bitpanda](https://www.bitpanda.com)
 - [Bitvavo](https://bitvavo.com)
```
Comment (lucas-tortora) [inline docs/get-started/introduction/iota/iota-token.md:82]:
```suggestion
- [Binance](https://www.binance.com)
- [Binance.US](https://www.binance.us/en/home)
- [Bitfinex](https://www.bitfinex.com/)
- [Bitpanda](https://www.bitpanda.com)
- [Bitvavo](https://bitvavo.com)
- [Gate.io](https://www.gate.io/)
- [HTX](https://www.htx.com/)
- [OKX](https://www.okx.com/)
```

Comment (lucas-tortora) [reply docs/get-started/introduction/iota/iota-token.md:82]: Let's alpha sort it :)

**Review by lucas-tortora — CHANGES_REQUESTED** [review]

File: docs/get-started/introduction/iota/iota-token.md (hunk 1 above)
Comment (vivekjain23) [reply docs/get-started/introduction/iota/iota-token.md]: Sure, Lucas.

**Review by lucas-tortora — APPROVED** [review]

**Comment by lucas-tortora** [comment]
@Ginowine and @salaheldinsoliman , could you please review this?

**Review by salaheldinsoliman — APPROVED** [review]
LGTM

**Pull request merged**